src/events/trigger-parsers/trigger-parser-base.ts

```diff
@@ -0,0 +1,21 @@
+import { IncomingEvent, Trigger } from '../event-data.types';
+
+export abstract class EventTriggerParser {
+  MAGIC_IDENTIFIER: string | null = null;
```

Comment:
still not a fan of the MAGIC, just saying :)

Reply:
whoops!
Fixed!